src/test/java/io/strimzi/ValidatingWebhookTest.java

```diff
@@ -54,13 +54,17 @@ public void setup() {
 
     @Test
     public void testEviction() {
-        String podName = "my-cluster-kafka-1";
-        when(podResource.get()).thenReturn(mockedPod(podName, true, false));
+        String podName = "my-cluster-foo";
```

Comment:
You can get rid of the `podName` parameter in `mockedPod` and `reviewRequest`. Simply set a constant like `my-cluster-kafka-1`.

You can also get rid of the `kindLabel` boolean parameter in `mockedPod`, because it's now part of the input map of labels.